src/main/defaults.ts

```diff
@@ -12,6 +12,7 @@ export const DEFAULT_OPTIONS: IMakeOptions = {
     target: 'apache',
     files: [],
     library: DEFAULT_APACHE_LIB,
+    strictUnions: false,
```

Comment:
Is the intention for `--strict` to potentially expand to enable additional strictness enhancements in the future (e.g. like `tsc`'s `--strict` flag)?

Just trying to understand why the flag is just called `--strict` but in code it's phrased more specifically as `strictUnions`.